doc_template/index.rst

```diff
@@ -35,19 +35,47 @@ The Allen SDK focuses on the Allen Brain Observatory, Cell Types Database, and M
 Allen Brain Observatory
 -----------------------
 
-The `Allen Brain Observatory <http://observatory.brain-map.org/visualcoding>`_ is a data resource for
-understanding sensory processing in the mouse visual cortex.  This study systematically measures visual
-responses in multiple cortical areas and layers using two-photon calcium imaging of GCaMP6-labeled neurons 
-targeted using Cre driver lines.  Response characterizations include orientation tuning, spatial and temporal
-frequency tuning, temporal dynamics, and spatial receptive field structure.
+The Allen Brain Observatory is a collection of data resources for understanding sensory 
+processing in the mouse visual cortex.  These resources systematically measure visual 
+responses in multiple cortical areas and layers using two-photon calcium imaging or 
+high-density extracellular electrophysiology (Neuropixels) probes.  Recordings are 
+performed on mice passively viewing visual stimuli or trained to actively perform an 
+image change detection task.
+
+.. list-table:: 
+   :widths: 25 50 50 25
+   :header-rows: 1
+
+   * - Behavior
+     - Modality
+     - Resource
+     - Initial Release
+   * - Passive
+     - Optical physiology
+     - `Visual Coding - Optical Physiology <https://allensdk.readthedocs.io/en/latest/brain_observatory.html>`_
+     - June 2016
+   * - Passive
+     - Extracellular electrophysiology
+     - `Visual Coding - Neuropixels <https://allensdk.readthedocs.io/en/latest/visual_coding_neuropixels.html>`_
+     - October 2019
+   * - Active
+     - Optical physiology
+     - Visual Behavior - Optical Physiology
```

Comment:
Now that we have the new page, this can now link to visual_behavior_ophys_nwb.rst